compiler/ballerina-lang/src/main/java/io/ballerina/projects/CodeGeneratorManager.java

```diff
@@ -297,6 +304,30 @@ public void addSourceFile(TextDocument textDocument, String filenamePrefix) {
             addSourceFile(textDocument, filenamePrefix, defaultModuleId);
         }
 
+        @Override
+        public void addTestFile(TextDocument textDocument, String filenamePrefix, ModuleId moduleId) {
```

Comment:
Let's rename this to `addTestSourceFile`

Reply:
Renamed via https://github.com/ballerina-platform/ballerina-lang/pull/35934/commits/f00062fd30e7ddbad3bcc66aeee8c2ec2b81275d